x-pack/package.json

```diff
@@ -27,6 +27,7 @@
     "**/@types/node": "10.12.27"
   },
   "devDependencies": {
+    "@cypress/webpack-preprocessor": "^4.1.0",
     "@kbn/dev-utils": "1.0.0",
```

Comment:
You also need to move:
```ts
"@types/js-yaml": "^3.12.1",
```
here

Reply:
Looks like that was added by someone else https://github.com/elastic/kibana/blob/d4832606ea35ebe949735695269248fbddae1337/x-pack/package.json#L67